libraries/AP_Baro/AP_Baro.h

```diff
@@ -147,6 +159,7 @@ class AP_Baro
     uint8_t _primary;
 
     struct sensor {
+        uint8_t type;                   // 0 for air pressure (default), 1 for water pressure
```

Comment:
Done, Done and Done! As always, I appreciate your attention to detail and quality feedback. Thank you.